### PR TITLE
Alter journey to use api gateway

### PR DESCRIPTION
**Pull request by fareed-mohammed**

### CODE REVIEW

File: app/views/v2/search-crn/search-crn.njk [hunk 1]
```diff
@@ -20,7 +20,40 @@
...
+                </span>
+              {% endfor %}
+            </ul>
+          </div>
+        </div>        
+      {% endif %}
+
+
+      <div class="govuk-form-group">
+        <h1 class="govuk-label-wrapper"><label class="govuk-label govuk-label--l" for="event-name">
```
Comment (johnwatson484) [inline app/views/v2/search-crn/search-crn.njk:43]: Maybe change the format of these input boxes to match the "asking multiple questions on a page" section here -> https://design-system.service.gov.uk/components/text-input/

**Review by johnwatson484 — COMMENTED** [review]
Suggestion about the format of the CRN/Caller ID page.

File: app/views/v2/search-crn/search-crn.njk (hunk 1 above)
Comment (fareed-mohammed) [reply app/views/v2/search-crn/search-crn.njk]: Changed headings

**Review by johnwatson484 — APPROVED** [review]
LGTM

**Pull request merged**